bedrock/firefox/templates/firefox/includes/download-button-thanks.html

```diff
@@ -0,0 +1,19 @@
+<div class="mzp-c-button-download-container c-button-download-thanks">
+  <a href="{{ transition_url }}"
+     id="{{ id }}"
+     class="mzp-c-button mzp-t-product {% if button_class %}{{ button_class }}{% else %}mzp-t-xl{% endif %}"
```

Comment:
It's not strictly needed, but I think having a singular identifiable class for the link (rather than needing `.c-button-download-thanks > a` makes sense. For compatibility purposes `download-link` wfm 👍